### PR TITLE
Create newsletter template entity test

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
**Description**
This PR with a test that creates Newsletter Template

**Steps:**
1 - Login as Admin
2 - Navigate to MARKETING > Newsletter Template
3 - Click the "Add New Template" button
4 - Fill in all data according to data set
5 - Save the template
6 - Making sure, that template was created

**Fixed Issues:**
#468 Convert CreateNewsletterTemplateEntityTest to MFT

### CODE REVIEW

**Comment by lbajsarowicz** [comment]
@DmitryTsymbal I know that's a minor issue, but could you lint your XML to remove the space before ` />`?

File: app/code/Magento/Newsletter/Test/Mftf/Section/NewsletterTemplateSection.xml [hunk 1]
```diff
@@ -7,6 +7,26 @@
...
+        <element name="searchResultFirstRow" type="text" selector=".data-grid>tbody>tr"/>
+        <element name="messageByType" type="block" selector="#messages .message-{{messageType}}" parameterized="true"/>
+    </section>
+    <section name="NewsletterMainActionsSection">
+        <element name="saveTemplateButton" type="button" selector=".page-actions-inner .page-actions-buttons .save"/>
+        <element name="deleteTemplateButton" type="button" selector=".page-actions-inner .page-actions-buttons .delete"/>
+        <element name="confirmDelete" type="button" selector=".action-primary.action-accept" timeout="10"/>
+    </section>
+    <section name="NewsletterMainActionsSection">
+        <element name="saveTemplateButton" type="button" selector=".save"/>
```
Comment (rogyar) [inline app/code/Magento/Newsletter/Test/Mftf/Section/NewsletterTemplateSection.xml]: Please, extend the simple selectors like `.save`, `.delete`. As I've already mentioned, there might be a case when we have two or more `.save` selectors on the page. In those cases, the test behavior will be unpredictable. That's why it's always a good practice to make the selectors more unique like `.page-actions-buttons .save`. So we are sure which `.save` element will be clicked.

Thank you!

Comment (DmitryTsymbal) [reply app/code/Magento/Newsletter/Test/Mftf/Section/NewsletterTemplateSection.xml]: Hello @rogyar , thanks for review.
For some reason I had double "NewsletterMainActionsSection" in the file with correct selectors from [MFTF-747](https://github.com/magento/magento-functional-tests-migration/pull/747) and previous version of these selectors.
So, I deleted previous one and left needed. Anyway, test works as expected for me.
Please review.
Thank you!

**Comment by tomreece** [comment]
I am processing this PR for delivery to mainline.
Please don't make any additional changes at this time because those commits may be missed and not merged correctly.

**Comment by DmitryTsymbal** [comment]
@soumyau , new PR was created https://github.com/magento/magento-functional-tests-migration/pull/761
Please check it.
Thanks

**Pull request closed without merging**